src/Orc.Automation.Recorder/Services/TreeBuilderService.cs

```diff
@@ -0,0 +1,100 @@
+﻿namespace Orc.Automation.Services;
+
+using System.Linq;
+using System.Windows;
+using System.Windows.Media;
+using AutomationMethods;
+using Catel.Reflection;
+
+public class TreeBuilderService
+{
+    public void Build(AutomationInformer informer)
+    {
+        var allActive = informer.Element.GetDescendants()
+            .Where(AutomationHelper.IsActiveModelControl)
+            .ToList();
+
+        var dataGrid = informer.Find(className: "Gum.Controls.DataGrid" + NameConventions.ActiveModelControlClassNameSuffix);
```

Comment:
This should not be in a public repository!

Reply:
wow

Reply:
right now it just a POCO...will deprecate this